testsuite/generate-format/tests/staged/consensus.yaml

```diff
@@ -224,6 +230,18 @@ Script:
     - args:
         SEQ:
           TYPENAME: TransactionArgument
+ScriptFunction:
+  STRUCT:
+    - module:
+        TYPENAME: ModuleId
+    - function:
+        TYPENAME: Identifier
+    - ty_args:
+        SEQ:
+          TYPENAME: TypeTag
+    - args:
+        SEQ:
+          TYPENAME: TransactionArgument
```

Comment:
@matbd would you mind commenting on whether the changes to the format yaml files are OK to you?

Reply:
LGTM